.github/ISSUE_TEMPLATE/release.yaml

```diff
@@ -0,0 +1,96 @@
+name:  Release
+description: Release/Pre-release of a version.
+labels: [release]
+body:
+- type: dropdown
```

Comment:
I added `Release Type`  and `Release Version` part in issue template, would there be other parts should be added from  `RELEASE.md`?